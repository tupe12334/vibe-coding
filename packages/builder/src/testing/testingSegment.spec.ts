import { describe, expect, test } from "vitest";
import { testingSegment } from "./testingSegment";

describe("testingSegment", () => {
  test("returns testing framework guidelines", async () => {
    const segment = await testingSegment("jest");
    expect(segment).toMatchSnapshot();
  });
});
