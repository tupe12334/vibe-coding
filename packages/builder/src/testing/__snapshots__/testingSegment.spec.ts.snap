// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`testingSegment > returns testing framework guidelines 1`] = `
[
  {
    "children": [
      {
        "type": "text",
        "value": "Testing framework",
      },
      {
        "type": "text",
        "value": " (jest)",
      },
    ],
    "depth": 2,
    "type": "heading",
  },
  {
    "children": [
      {
        "children": [
          {
            "children": [
              {
                "type": "text",
                "value": "Use Jest for fast and comprehensive unit testing.",
              },
            ],
            "type": "paragraph",
          },
        ],
        "type": "listItem",
      },
    ],
    "ordered": false,
    "type": "list",
  },
]
`;
